source/includes/gho/_api.md

```diff
@@ -1118,7 +1113,7 @@ customFieldValues | [\[CustomFieldValue\]](#customfieldvalue) |
 dateOfBirth | [Date](#date) |
 department | [Department](#department) |
 email | [String](#string) |
-employmentStatus | [String](#string) |
+employmentStatus | [String](#string) | [Valid options](#employmentstatuses-string)
```

Comment:
Is it overkill to add this here? Nothing else has a description.

![image](https://user-images.githubusercontent.com/56364494/102514784-e9485c80-405a-11eb-908e-42e7b44da5ae.png)